src/pages/login.js

```diff
@@ -0,0 +1,122 @@
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useForm, Controller } from 'react-hook-form';
+import Navbar from './components/Navbar';
+import Input from './components/Input';
+import { PORT } from '../siteConstants';
+
+const Login = () => {
+  const { t } = useTranslation('login');
+  const [buttonText, setButtonText] = useState('Login');
+  const [errorMessage, setErrorMessage] = useState('');
+  const [isButtonDisabled, setIsButtonDisabled] = useState(false);
+  const [igboApiRoute, setIgboApiRoute] = useState('');
+  const { handleSubmit, control, errors, reset } = useForm();
+
+  useEffect(() => {
+    const productionApiRoute = 'https://igboapi.com';
+    const developmentApiRoute = `http://localhost:${PORT}`;
+    let apiRoute = developmentApiRoute;
+    if (typeof window !== 'undefined') {
+      apiRoute =
+        window.location.hostname === 'igboapi.com' && process.env.NODE_ENV === 'production'
+          ? productionApiRoute
+          : developmentApiRoute;
+    } else {
+      apiRoute = process.env.NODE_ENV === 'production' ? productionApiRoute : developmentApiRoute;
+    }
+    setIgboApiRoute(apiRoute);
+  }, []);
```

Comment:
`useEffect` should right above the `return` statement for the functional component and after all `const` variables and functions

Reply:
done

Reply:
the useEffect was removed, since there is no route for it on the backend yet.